### ISSUE TITLE
wxPython 4.1.1 - FlatNotebook crashes on Linux when using tab style FNB_VC8

### ISSUE THREAD

**Issue by suurjaak** [wxWidgets/Phoenix#1904]
**Operating system**: Ubuntu 20.04
**wxPython version & source**: 4.1.1 gtk3 (phoenix) wxWidgets 3.1.5 (pip-installed)
**Python version & source**: python 2.7.18 64bit (distro)

**Description of the problem**:

- run wxPython demo
- select FlatNotebook
- click "Test FlatNotebook"
- menu Edit -> New Page
- menu Tab Appearance -> Tabs Style -> Use VC8 Style

Result: program crashes (segmentation fault). 

Same result in Ubuntu 18.04. 

No crash with wxPython 4.1.0 gtk3 (phoenix) wxWidgets 3.1.4.

**Comment by suurjaak** [comment]
Minimal program to reproduce:
```
import wx
import wx.lib.agw.flatnotebook as fnb

app = wx.App()
frame = wx.Frame(None)

book = fnb.FlatNotebook(frame, agwStyle=fnb.FNB_VC8)
book.AddPage(wx.Panel(book), "Page")

frame.Show()
app.MainLoop()
```

**Comment by swt2c** [comment]
This appears to be crashing in the wx.Region constructor, so I think duplicate of #1878, should be fixed in latest snapshots.

**Comment by swt2c** [comment]
Confirmed this was fixed by the fix I did for #1878.
